### PR TITLE
project: try alexellis/derek bot to manage PRs and issues

### PR DESCRIPTION
**Pull request by tiborvass**
**- What I did**
Added support for @alexellis's derek bot, which extends Github's user permissions to help non-committers to manage issues and pull requests simply by commenting.
More information on https://github.com/alexellis/derek#derek

**- How I did it**
Added a .DEREK file and updated CONTRIBUTING.md

**- How to verify it**

**- Description for the changelog**
Experiment using alexellis/derek bot to manage PRs and issues

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -311,6 +311,36 @@ Don't forget: being a maintainer is a time investment. Make sure you
...
+`.DEREK.yml` file. [Derek](https://github.com/alexellis/derek) is a bot that extends
+Github's user permissions to help non-committers to manage issues and pull requests simply by commenting.
+
+For example:
+
+* Labels
+
+```
+Derek add label: duplicate
+Derek remove label: good first issue
```
Comment (thaJeztah) [inline CONTRIBUTING.md]: perhaps use actual labels from this repo here

**Review by thaJeztah — APPROVED** [review]
small nit, but LGTM otherwise

**Comment by tiborvass** [comment]
fixed

**Review by yongtang — APPROVED** [review]
LGTM

**Review by justincormack — APPROVED** [review]

**Comment by thaJeztah** [comment]
Let's merge this!

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Comment by justincormack** [comment]
Would be nice at some point to have a shorter syntax for rebuild, which is currently `add label: rebuild/janky` and somewhat indirect as a different bot handles that!

**Comment by alexellis** [comment]
Great to see this merged. @thaJeztah maybe we can find 5-10 minutes to talk about it on the next maintainers zoom call / captains briefing? @justincormack would like to know more about that use-case.

**Comment by alexellis** [comment]
Derek add label: area/project

**Comment by alexellis** [comment]
Derek lock

**Comment by thaJeztah** [comment]
derek unlock

**Comment by thaJeztah** [comment]
Derek unlock

**Comment by alexellis** [comment]
It works 👍

**Comment by dmcgowan** [comment]
:angry:

**Pull request merged**